librz/util/subprocess.c

```diff
@@ -125,6 +125,19 @@ static LPWCH override_env(const char *envvars[], const char *envvals[], size_t e
 	return ret;
 }
 
+static void remove_cr_on_cond(char *str) {
```

Comment:
what does this name mean exactly? I'm not sure I understand what the function is supposed to do based on the name.

Reply:
`remove_cr_on_condition`, but I think I should just shorten it to `remove_cr`

Reply:
or maybe I should just rename it back to `remove_cr_from_crlf`

Reply:
5adf82014edf99fd428e379b4c3b66063951e800 ok renamed the function ... I chose `remove_cr` because it more accurately describes what the function is doing right now.